cdmtaskservice/models.py

```diff
@@ -207,6 +207,14 @@ class Parameter(BaseModel):
         min_length=1,
         max_length=1000,
     )] = None
+    container_num_prefix: Annotated[ArgumentString | None, Field(
+        example="/output/job-",
```

Comment:
Does `example` work as expected for you? My PyCharm flags warning for this field. I assumed it worked for you. But I looked at `field` API doc today and noticed `example` is not a valid parameter instead it's `examples`.
https://docs.pydantic.dev/2.10/api/fields/

Reply:
> Does example work as expected for you?

Yes, and `examples` didn't as of December of last year and hadn't since Collections days: https://github.com/kbase/cdm-task-service/blob/main/cdmtaskservice/models.py#L28-L30

That being said, I just tried again and `examples` finally works, so in a future PR I'll update all of them at once